modules/library/jdbc/src/main/java/org/geotools/jdbc/SQLDialect.java

```diff
@@ -384,6 +385,16 @@ public void registerAggregateFunctions(Map<Class<? extends FeatureVisitor>,Strin
     }
     
     /**
+     * Register the dialect mappings between Geotools FilterFunction names and database
+     * related function.
+     * 
+     * @param functions mappings from GT FilterFunction name to sql function name
+     */
+    public void registerFunctions(Map<String ,String> functions) {
```

Comment:
This is a new public method for SQLDialect opening up an ability to extend for client applications; do we need a proposal here to consider this API change?